suites/pacific/cephadm/test_5-2_new-feature_apply-spec.yaml

```diff
@@ -0,0 +1,128 @@
+#===============================================================================================
```

Comment:
can we have one test suite which covers CLI and apply-spec deployment and let regression test suites be executed in other stages.